### PR TITLE
Added a very simple PG in Physics Forces

### PR DESCRIPTION
**Pull request by flushpot1125**
Hi core great contributors!
I learned Havok plugin and found no simple examples in "Forces" section.

My idea is that showing simple PGs helps for BJS beginners.

I'm very glad if you review my added PGs and any feedback would welcome. Thanks for taking your time.

### CODE REVIEW

**Comment by sebavan** [comment]
thanks, just waiting for some of our physics ppl to review before merging :-)

File: content/features/featuresDeepDive/physics/v2/forces.md [hunk 1]
```diff
@@ -43,7 +43,13 @@ A force is a continuous effect that is applied to an object over time, which can
 
 An impulse, on the other hand, is a sudden, instantaneous effect that changes the velocity of an object. It is a specific amount of force applied over a very short duration of time, often modeled as a single frame in a game. For example, a collision between two objects might generate an impulse that changes the direction and speed of both objects.
 
-In summary, a force is a *continuous* effect over time, while an impulse is a sudden, *instantaneous* effect that changes the velocity of an object.
+In summary, a force is a *continuous* effect over time, while an impulse is a sudden, *instantaneous* effect that changes the velocity of an object.  
+
+<Playground id="#R66K4K" title="applyForce" description="Show how to use applyForce in a very simple example" isMain={true} category="Physics"/>
+
+<Playground id="#Z8HTUN#671" title="applyImpulse" description="Show how to use applyImpulse in a very simple example" isMain={true} category="Physics"/>
```
Comment (RaananW) [inline content/features/featuresDeepDive/physics/v2/forces.md]: isMain={true} is a flag we are adding to specific playgrounds and not all. Would be great if you could remove it :-)

```suggestion
<Playground id="#R66K4K" title="applyForce" description="An example of using applyForce" category="Physics"/>

<Playground id="#Z8HTUN#671" title="applyImpulse" description="An example of using applyImpule" category="Physics"/>
```

File: content/features/featuresDeepDive/physics/v2/forces.md [hunk 2]
```diff
@@ -43,7 +43,13 @@ A force is a continuous effect that is applied to an object over time, which can
 
 An impulse, on the other hand, is a sudden, instantaneous effect that changes the velocity of an object. It is a specific amount of force applied over a very short duration of time, often modeled as a single frame in a game. For example, a collision between two objects might generate an impulse that changes the direction and speed of both objects.
 
-In summary, a force is a *continuous* effect over time, while an impulse is a sudden, *instantaneous* effect that changes the velocity of an object.
+In summary, a force is a *continuous* effect over time, while an impulse is a sudden, *instantaneous* effect that changes the velocity of an object.
```
Comment (RaananW) [inline content/features/featuresDeepDive/physics/v2/forces.md]:
```suggestion
In summary, a force is a *continuous* effect over time, while an impulse is a sudden, *instantaneous* effect that changes the velocity of an object. Click on the sphere in the following examples to experiment with forces and impulses
```

**Review by RaananW — CHANGES_REQUESTED** [review]
Just a few working suggestions and a small request to remove the isMain

File: content/features/featuresDeepDive/physics/v2/forces.md [hunk 3]
```diff
@@ -43,7 +43,13 @@ A force is a continuous effect that is applied to an object over time, which can
 
 An impulse, on the other hand, is a sudden, instantaneous effect that changes the velocity of an object. It is a specific amount of force applied over a very short duration of time, often modeled as a single frame in a game. For example, a collision between two objects might generate an impulse that changes the direction and speed of both objects.
 
-In summary, a force is a *continuous* effect over time, while an impulse is a sudden, *instantaneous* effect that changes the velocity of an object.
+In summary, a force is a *continuous* effect over time, while an impulse is a sudden, *instantaneous* effect that changes the velocity of an object. Click on the sphere in the following examples to experiment with forces and impulses
+
+<Playground id="#R66K4K" title="applyForce" description="An example of using applyForce" category="Physics"/>
```
Comment (carolhmj) [inline content/features/featuresDeepDive/physics/v2/forces.md]:
```suggestion
<Playground id="#R66K4K#1" title="applyForce" description="An example of using applyForce" category="Physics"/>
```

File: content/features/featuresDeepDive/physics/v2/forces.md [hunk 4]
```diff
@@ -43,7 +43,13 @@ A force is a continuous effect that is applied to an object over time, which can
 
 An impulse, on the other hand, is a sudden, instantaneous effect that changes the velocity of an object. It is a specific amount of force applied over a very short duration of time, often modeled as a single frame in a game. For example, a collision between two objects might generate an impulse that changes the direction and speed of both objects.
 
-In summary, a force is a *continuous* effect over time, while an impulse is a sudden, *instantaneous* effect that changes the velocity of an object.
+In summary, a force is a *continuous* effect over time, while an impulse is a sudden, *instantaneous* effect that changes the velocity of an object. Click on the sphere in the following examples to experiment with forces and impulses
+
+<Playground id="#R66K4K" title="applyForce" description="An example of using applyForce" category="Physics"/>
+
+<Playground id="#Z8HTUN#671" title="applyImpulse" description="An example of using applyImpule" category="Physics"/>
```
Comment (carolhmj) [inline content/features/featuresDeepDive/physics/v2/forces.md]:
```suggestion
<Playground id="#Z8HTUN#677" title="applyImpulse" description="An example of using applyImpule" category="Physics"/>
```

**Review by carolhmj — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just some suggestions relative to the PGs:
- I switched the force PG to apply the force in the Y-axis, so it's consistent with the impulse PG
- When applying forces/impulses, it's safer to use `absolutePosition` instead of `position`, because the body can be parented. It doesn't make a difference in this case, but it's nice to show the safer way :smile:

File: public/img/playgroundsAndNMEs/pg-R66K4K.png
Comment (RaananW) [inline public/img/playgroundsAndNMEs/pg-R66K4K.png:1]: if the playgrounds were changed, those two images are not needed (and will be generated in the next build process). They can also be renamed (add `-1` for this filename, and change `671` to `672` in the other

**Review by RaananW — APPROVED** [review]
my changes were merged, resetting mine.
In general - LGTM, just a small comment regarding the new playgrounds

**Comment by carolhmj** [comment]
Added my changes and removed the images @RaananW it's ready to merge

**Review by carolhmj — APPROVED** [review]

**Review by sebavan — APPROVED** [review]

**Comment by flushpot1125** [comment]
Thanks for a lot of supporting!!!
